lizard.py

```diff
@@ -688,6 +688,13 @@ def print_warnings(option, scheme, warnings):
     return warning_count, warning_nloc
 
 
+def print_no_warnings(option):
+    warn_str = "No thresholds exceeded ({0})".format(
+        ' or '.join("{0} > {1}".format(
+            k, val) for k, val in option.thresholds.items()))
```

Comment:
Code formatting here is a bit confusing.
Why not move the second ``format`` ``k, vall)`` to the same line?